CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
Let's revisit the idea once we surpass 80+ commits between releases (without merges).

:+1: for the `/ unreleased` changelog in general.